README.md

```diff
@@ -1 +1,78 @@
-# netper
\ No newline at end of file
+# Network Performance
+
+This repo hosts a shell script that will run various network performance measurements using various configuration 
```

Comment:
I suggest to write it this way "This repo hosts a shell script that will run various network performance measurements using variable configuration"